code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router'
+
+import { DETAIL_API, BACKDROP_PATH, POSTER_PATH } from '../reusables/urls'
+
+import { BackButton } from '../components/BackButton'
+
+export const MovieDetails = () => {
+  const [movieDetails, setMovieDetails] = useState([])
+
+  const { id } = useParams()
+
+  useEffect(() => {
+    fetch(DETAIL_API(id))
+      .then(res => res.json())
+      .then(json => setMovieDetails(json))
+      .catch(err => console.err(err))
+  },[id])
+
+  return (
+    <>
+      <BackButton />
+      {movieDetails.length !== 0 && ( 
+        <>
```

Comment:
Since MovieDetails are displayed on a new page, we would suggest using section here as you did in MovieList to make it more semantic :)